tests/arp/test_neighbor_mac_noptf.py

```diff
@@ -69,9 +69,10 @@ def setupDutConfig(self, duthosts, enum_rand_one_per_hwsku_frontend_hostname):
                 None
         """
         duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
-        duthost.command("sudo config bgp shutdown all")
-        if not wait_until(120, 2.0, self._check_no_bgp_routes, duthost):
-            pytest.fail('BGP Shutdown Timeout: BGP route removal exceeded 120 seconds.')
+        if not duthost.get_facts().get("modular_chassis"):
+            duthost.command("sudo config bgp shutdown all")
+            if not wait_until(120, 2.0, self._check_no_bgp_routes, duthost):
```

Comment:
can _check_no_bgp_routes be updated for VOQ chassis to check on eBGP routes and skip other routes ?

Reply:
thanks !!